WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -638,22 +638,27 @@ public void run() {
             mCallback.onAdapterSelectionCountChanged(mSelectedItems.size());
         }
 
-        PhotoPickerUtils.announceSelectedImageForAccessibility(holder.mImageView, selected);
+        PhotoPickerUtils.announceSelectedImageForAccessibility(holder.mImageView, isVideo, selected);
```

Comment:
Though not introduced in this PR, I wonder if we can rename this method to `announceSelectedItemForAccessibility`, or `announceSelectedMediaForAccessibility`, or something like that, since it could be video and not only images. Wdyt?

Reply:
Great point @mkevins 👍 
Renamed with cfe595ddafbf0966b9b9164848c4cfeeff8812e7

Reply:
That was fast!